README.md

```diff
@@ -40,60 +98,40 @@ BIFROST_MANAGEMENT_API_KEY=<< provided on sign up >>
-
-### Search Content management
-
-The [silverstripe/silverstripe-forager](https://github.com/silverstripeltd/silverstripe-forager) module contains provider-agnostic interfaces and admin UI for configuring the service and index management. The [silverstripe/silverstripe-forager-bifrost](https://github.com/silverstripeltd/silverstripe-forager-bifrost) contains Silverstripe Search specific implementation to connect you with our Bifröst API. 
+> [!TIP]
+> For Silverstripe Cloud clients:
+> - `BIFROST_ENDPOINT` and `BIFROST_ENGINE_PREFIX` should be added as Variables
+> - `BIFROST_MANAGEMENT_API_KEY` and `BIFROST_QUERY_API_KEY` should be added as Secrets.
 
-Refer to the [silverstripe/silverstripe-forager docs](https://github.com/silverstripeltd/silverstripe-forager) for detailed information.
+### Understanding your engine prefix and suffix:
 
-### Search UI
+* All Silverstripe Search engine names follow a 4 slug format like this: `search-<subscription>-<environment>-<suffix>`
+* Your `<engine-prefix>` is everything except `-<suffix>`; so, it's just `search-<subscription>-<environment>`
 
-The [silverstripe/silverstripe-discoverer-search-ui](https://github.com/silverstripeltd/silverstripe-discoverer-search-ui) module contains the base parts to show results on a Search page including a Page Type, Controller, templates, and basic styling.
+For example:
 
-### Search page and results
+| Engine                    | Engine prefix        | Engine suffix |
+|---------------------------|----------------------|---------------|
+| search-acmecorp-prod-main | search-acmecorp-prod | main          |
+| search-acmecorp-prod-inc  | search-acmecorp-prod | inc           |
+| search-acmecorp-uat-main  | search-acmecorp-uat  | main          |
+| search-acmecorp-uat-inc   | search-acmecorp-uat  | inc           |
 
-The [silverstripe/silverstripe-discoverer-search-ui](https://github.com/silverstripeltd/silverstripe-discoverer-search-ui) package contains a base Search Page Type, Controller and Templates that work with the [discoverer](https://github.com/silverstripeltd/silverstripe-discoverer) and [forager](https://github.com/silverstripeltd/silverstripe-forager) modules. You can use the CMS to add the new Page Type to get up and running quickly.
+**Why?**
 
-To customise the look and feel of your search page then check out the [silverstripe/silverstripe-discoverer-search-ui](https://github.com/silverstripeltd/silverstripe-discoverer-search-ui) docs. If you would like to add filters or other advance query options then head over to the [discoverer](https://github.com/silverstripeltd/silverstripe-discoverer) module. 
+Because you probably have more than one environment type that you're running search on (e.g. Production and UAT), and (generally speaking) you should have different engines for each of those environments. So, you can't just hardcode the entire engine name into your project, because that code doesn't change between environments.
```

Comment:
maybe this explanation should live on forager/discoverer as its useful there too?

Reply:
Yup, I've duplicated a bunch of this stuff in those 2 modules (in fact, these definitions came from those modules).